strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/nuget/NugetPackageControllerTest.java

```diff
@@ -395,4 +397,23 @@ public void testRemoteProxyGroup()
         }
     }
 
+    @Test
+    public void testRemoteLastVersion()
+            throws Exception
+    {
+        given().header("User-Agent", "NuGet/*")
+               .when()
+               .get(getContextBaseUrl() + "/storages/public/nuget-public/FindPackagesById()?id=NHibernate&$orderby=Version")
+               .then()
+               .statusCode(HttpStatus.OK.value())
+               .and()
+               .assertThat()
+               .body("feed.title", equalTo("Packages"))
+               .and()
+               .assertThat()
+               .body("feed.entry[0].title", equalTo("NHibernate"))
+               .body(hasXPath("/atom:feed/atom:entry[last()]/m:properties/d:IsLatestVersion[text()='true']"));
```

Comment:
@carlspring @fuss86 @steve-todorov 
guys, do anyone knows XPAth?
Need help to make this expression work :)